languages/python/README.md

```diff
@@ -31,5 +32,8 @@ pip install bitwarden-sdk
 Set the `ORGANIZATION_ID` and `ACCESS_TOKEN` environment variables to your organization ID and access token, respectively.
 
 ```bash
+source .venv/bin/activate
```

Comment:
Moved this to a Using Virtual Environments section away from run. If anything, it will hopefully help those who are less familiar with Python get started.

[d2b769a](https://github.com/bitwarden/sdk/pull/592/commits/d2b769ad70872190f54de962c8c8d629b1df40ae)